### DISCUSSION TITLE
Welcome to Zikula Discussions!

### DISCUSSION THREAD

**Discussion by craigh** [zikula/core#4561] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by markcjf** [comment]
not a very busy place this!!!
tried installing the latest version of zikula (3.1.0) but kept failing - decided to install the older version (3.0.4) and that install no problem so tried to upgrade - I can't see an upgrade path just a notice that there is a new version and to download the zip. so copied files over to installed version and got the same 500 error - so restored older version again and fine. When I tried to install through command line got 5BMouth or some such thing being not defined ( The parameter "5BMouth" must be defined. /var/www/zikula$ In KernelProdContainer.php line 2904 )
so - help!

**Reply by craigh** [reply]
- nope. not busy.
 - replying to an existing thread is not a good way to get help. Create your own threads.
 - `5BMouth` is not related to zikula and [does not exist in the entire organization](https://github.com/search?q=org%3Azikula+5BMouth). So we cannot help with that.

**Reply by Guite** [reply]
You can just stay with 3.0.x unless there is a real killer feature in 3.1 you've waited for; there won't be an upgrade path to 4.0 (because it changes everything) so you could just forget 3.1.
